### ISSUE TITLE
Error on run

### ISSUE THREAD

**Issue by shelbyKiraM** [b-adams01/Scriptable-Lock-Screen-Widgets#1]
I could prob find it in some other repo of yours, but can you put this file in this repo?
> `Error on line 86:26: No file to import at no-background.js.`

**Comment by b-adams01** [comment]
I added the [no-background.js](https://github.com/b-adams01/Scriptable-Lock-Screen-Widgets/blob/main/no-background.js) to this repo.

**Comment by b-adams01** [comment]
I added the [no-background.js](https://github.com/b-adams01/Scriptable-Lock-Screen-Widgets/blob/main/no-background.js) to this repo.

**Comment by shelbyKiraM** [comment]
Ahhh that's awesome, but when trying to put Battery on Lock Screen widget, I got an error about not able to display alerts in widgets, also, I'm an iPhone 14 Pro Max, when I try to pick a home screen screenshot it fails. However, when I commented out the nobg lines it just worked. For LS Battery. LS Volume fails with "Session activation failed" on the static method
```js
Device.volume()
             ^
```
running from the Lock Screen but it comes up fine in the app with:
```js
// widget.presentSmall();
widget.presentAccessoryCircular();
```

buuuut... additionally, I'm finding it hard to differentiate between the widgets on my LS. I tweaked the weather circles widget to only show the temp circle with a thermometer in the middle like this:
![image](https://user-images.githubusercontent.com/202692/204442809-85979fe4-eb27-40fa-8a24-42122b5aa6eb.png)
(reminds me, gonna put a ° sign after temp)
